app/assets/stylesheets/components/_published-dates.scss

```diff
@@ -1,16 +1,26 @@
 .app-c-published-dates {
-  @include core-16;
```

Comment:
Again, you could try `govuk-link` on this element. I'm going to stop writing comments on this because there's a lot of them, the main benefit is that with the additional link class we're not duplicating the focus styles every time the mixin is called.